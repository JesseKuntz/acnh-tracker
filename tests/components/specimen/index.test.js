import { h } from 'preact';
import Specimen from '../../../src/components/specimen';

import { shallow } from 'enzyme';

describe('Specimen', () => {
  test('matches snapshot', () => {
    expect(
      shallow(<Specimen type={'type'} image={'image'} name={'name'} />)
    ).toMatchSnapshot();
  });
});
