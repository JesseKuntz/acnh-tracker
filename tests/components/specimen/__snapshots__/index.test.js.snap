// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Specimen matches snapshot 1`] = `
<div
  className="specimen-base specimen-tracker"
  onClick={[Function]}
>
  <picture>
    <source
      srcset="../../assets/type/webp/image.webp"
      type="image/webp"
    />
    <source
      srcset="../../assets/type/png/image.png"
      type="image/png"
    />
    <img
      className="logo"
      src="../../assets/type/png/image.png"
    />
  </picture>
  <div
    className="textContainer"
  >
    <p
      className="text"
    >
      name
    </p>
  </div>
</div>
`;
